eeg_rishikesh/sub-001/ses-01/eeg/sub-001_ses-01_task-meditation_channels.tsv

```diff
@@ -76,5 +76,5 @@ GSR2	GSR	n/a
 Erg1	MISC	n/a
 Erg2	MISC	n/a
 Resp	RESP	n/a
-Plet	PPG	n/a
+Plet	PD	n/a
```

Comment:
are you sure this is a photodiode @rwblair ? also tagging @arnodelorme to make him aware that the meditation dataset needs an update because the PPG "type" is not a valid type.

What kind of channel was that Arno?

Reply:
A PPG is not simply a photodiode. For a PD (as per [IEEG specification](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/04-intracranial-electroencephalography.html#channels-description-_channelstsv)) I would assume that the channel represents the signal of a photodiode that is attached to the corner of a screen. That is common in clinical iEEG experiments in EMUs (Epilepsy Monitoring Units), where screens used for stimuli are often not precisely calibrated.

Would it not be better to add `PPG` as an allowed type? I would say that a PPG channel is unambiguous. We also use them often for physio recordings during fMRI sessions.

Reply:
I have never used `PPG` but I am happy to follow your and @arnodelorme's  lead here @robertoostenveld 

https://github.com/bids-standard/bids-specification/issues/561

A Pull Request is welcome, I'll be on vacation starting from tomorrow though.